openeo-geotrellis/src/main/scala/org/openeo/geotrellis/OpenEOProcesses.scala

```diff
@@ -880,6 +880,27 @@ class OpenEOProcesses extends Serializable {
     ContextRDD(filtered,data.metadata.copy(bounds = newBounds))
   }
 
+  def transfromSparceSpaceTimePartition(data: MultibandTileLayerRDD[SpaceTimeKey], target: MultibandTileLayerRDD[SpaceTimeKey]):Option[SparseSpaceTimePartitioner]= {
+    val index = data.partitioner.get.asInstanceOf[SpacePartitioner[SpaceTimeKey]].index.asInstanceOf[SparseSpaceTimePartitioner]
```

Comment:
@ElienVandermaesenVITO  I think  index.asInstanceOf[SparseSpaceTimePartitioner] will throw an exception if index is not the right type, which is very much possible. Please add extra if statement here to first check for the right type.